bundles/org.openhab.binding.freeboxos/src/main/resources/OH-INF/config/repeater-config.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:config-description="https://openhab.org/schemas/config-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/config-description/v1.0.0
+		https://openhab.org/schemas/config-description-1.0.0.xsd">
+
+	<config-description uri="thing-type:freeboxos:repeater">
+		<parameter name="refreshInterval" type="integer" min="1" required="true" unit="s">
+			<label>Refresh Interval</label>
+			<description>The refresh interval in seconds which is used to poll the repeater</description>
+			<default>30</default>
+		</parameter>
+		<parameter name="macAddress" type="text" pattern="([0-9A-Fa-f]{2}[:-]){5}([0-9A-Fa-f]{2})" required="true">
+			<label>MAC Address</label>
+			<description>The MAC address of the network device</description>
+		</parameter>
+		<parameter name="id" type="integer" required="true">
+			<label>ID</label>
+			<description>Id of the repeater</description>
+			<default>1</default>
```

Comment:
Does it make sense to define a default id ?
In this case, please remove the tag "required".
But it makes probably more sense to keep the tag "required" and remove the default value.

Reply:
The fact is that this default value has great chance to be good, but you're right, it's a high probability value, but still a guess.

Reply:
Ok, keep the default value.